versioned_docs/version-0.8/Installation/updating.md

```diff
@@ -27,9 +37,19 @@ sudo chmod -R 755 /var/www/controlpanel
 
 ### Update Dependencies
 
+:::caution
+Please delete your old vendor folder before running the composer command.
+```bash
+sudo rm -rf /var/www/controlpanel/vendor
+```
+:::
+
+:::info
+Make sure composer actualy uses php 8.3 and not longer 7.4!
```

Comment:
actualy -> actually
not longer 7.4 -> not 7.4

nitpicky - "Composer now uses PHP 8.3 and not 7.4!"